scripts/integrations/angular/src/app/binding/editor-binding.component.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import {
+	Component,
+	ViewChild,
+	AfterViewInit
+} from '@angular/core';
+
+@Component( {
+	selector: 'app-binding',
+	templateUrl: './editor-binding.component.html'
+} )
+export class EditorBindingComponent implements AfterViewInit {
+	@ViewChild( 'editor' ) editor;
+
+	public editorData = '<p>This is a CKEditor 4 WYSIWYG editor instance created with Angular.</p>'
```

Comment:
Angular version of this component has a different (richer) toolbar than the React and Vue ones.